frontend/src/components/teacher/student-management/classrooms/AddClassroomModal.tsx

```diff
@@ -92,26 +105,30 @@ const AddClassroomModal = ({
 
   const onModalClose = () => {
     setValue("className", "");
-    setValue("schoolYear", "");
+    setValue("startDate", undefined);
     setValue("gradeLevel", Grade.K);
     setShowRequestError(false);
     setRequestErrorMessage("");
     onClose();
   };
 
   const onSave: SubmitHandler<ClassroomForm> = async (data) => {
+    const dates = watch("startDate");
     if (!validateFields()) {
       setShowRequestError(true);
       setRequestErrorMessage(
         "Please ensure all required components are filled out before saving changes",
       );
+    } else if (dates && dates < new Date() && !isSameDay(dates, new Date())) {
```

Comment:
nit: would be better to use the same date for both checks (i.e. make a constant `now` above).